conanfile.py

```diff
@@ -76,8 +76,9 @@ class BasiliskConan(ConanFile):
 
     # Requirements
     requires = [
-        "eigen/3.4.0",
-        "cspice/0067",
+        # Core dependencies with explicit version ranges
+        "eigen/[>=3.4.0 <4.0.0]",
```

Comment:
3.4.0 is the tastes, just make it use 3.4.0 for now

Reply:
Actually, with conan we are specifying specific versions of conan package.  No need to enter a range of suitable conan packages.  We can remove these edits.   

Patrick was talking about pining a specific build ID number for each package.  I don't see a need for this.